tensorflow/contrib/cmake/tools/create_def_file.py

```diff
@@ -64,7 +64,7 @@
                         r"tensorflow::|"
                         r"functor::|"
                         r"\?nsync_|"
-                        r"perftools::gputools")
+                        r"stream_executor::")
```

Comment:
This looks like the right fix, but it seems we're once again hitting the 2**16 exported symbols limit. Perhaps `stream_executor::*` contains far more symbols that `perftools::gputools` ever did?

I don't know anything about the structure of that namespace, but is it possible we could use a narrower regex to get the symbols we need here? Ideally we'd have some sort of explicit `public` or `private`/`impl` subnamespace that we could include/exclude as possible, but I'm not sure if such a thing exists...